tensorflow/lite/tools/delegates/delegate_provider.h

```diff
@@ -157,6 +157,9 @@ class ProvidedDelegateList {
     return CreateAllRankedDelegates(*params_);
   }
 
+  // Returns the count of referenced delegates in providers_ variable
+  const int GetCount();
```

Comment:
Might simply inline the implementation here? Btw, if the change to tensorflow/lite/kernels/test_delegate_providers.cc were dropped, this function would be no longer needed?

Reply:
Implementation has been inlined in the latest commit. The answer is yes. However I would suggest keeping it for the reasons discussed above.